src/store/FullScreenDialogStore/FullScreenDialogStore.ts

```diff
@@ -0,0 +1,44 @@
+import { ElementType } from 'react';
+import { action, makeObservable, observable } from 'mobx';
+import { openDialogTypes } from '@store/FullScreenDialogStore/FullScreenDialogStore.types';
+
+class FullScreenDialogStore {
+  header?: string;
+  isOpened$ = false;
+  textOfSubmit$ = '';
+  elementToRender?: ElementType;
+  defaultValuesForm: object = {};
+  propsOfElement: object = {};
+  constructor() {
+    makeObservable(this, {
+      textOfSubmit$: observable,
+      isOpened$: observable,
+      elementToRender: observable,
+      propsOfElement: observable,
+      openDialog: action,
+      closeDialog: action
+    });
+  }
+  openDialog({
+    defaultValuesForm,
+    element,
```

Comment:
you can forward element as <Element props={props} />
no need to pass propsOfElement here